src/form/field/FormField.tsx

```diff
@@ -37,13 +37,21 @@ function FormField(props: FormFieldProps) {
 
       {hasErrorMessage &&
         errorMessages?.map((message) => (
-          <FormFieldMessageRow type={"error"} message={message} />
+          <FormFieldMessageRow
+            key={`error.${message}`}
+            type={"error"}
+            message={message}
+          />
         ))}
```

Comment:
I think it should be wrapped with `List` component. Same for helper messages below as well.